.github/workflows/ci.yml

```diff
@@ -497,9 +497,14 @@ jobs:
       - if: ${{ matrix.target == 'Web' && matrix.browser != 'webkit' }}
         run: 'yarn workspace @getodk/web-forms test-browser:${{ matrix.browser }}'
 
+      - if: ${{ matrix.target == 'Web' }}
+        name: 'Run functional e2e tests'
+        run: 'yarn workspace @getodk/web-forms test:e2e:functional:${{ matrix.browser }}'
+
       # xvfb-run is used for visual tests to compare snapshots more accurately.
       - if: ${{ matrix.target == 'Web' }}
```

Comment:
Ultimately I see this as being a separate job. This will make it easier to re-run if there's flakiness, and during development, verify that functionally it's working but you need to update the screenshots. But we can leave this as a step for another day.